### ISSUE TITLE
Notification : unknown error

### ISSUE THREAD

**Issue by foxmask** [wallabag/android-app#314]
when the network is lost and the sync was in progress we should not have to see the "unknown error" .
I think people are asking themselve what's wrong as they wont know from where does this come from.

![unknown_error](https://cloud.githubusercontent.com/assets/208212/19093412/62c35978-8a8b-11e6-9ff9-942e46f17b60.png)

**Comment by di72nn** [comment]
Thanks for your report. We try to improve it as part of #304. I created #316 that is likely to cover your issue.

Please report if you encounter similar errors (not included in the list in #304). We might need the exception info from the extended notification text (as noted in #304).

**Comment by nsteinmetz** [comment]
Hi,

I have the same error, even with 1.11.0 release and wallabag 2.1.2

I have it both on the wifi or on 4G networks.

Let me know if we can provide you more verbose logs...

Nicolas

**Comment by tcitworld** [comment]
What's the wallabag server version ? You can provide us a logcat if you have Android Studio installed or your phone rooted with a logcat app installed.

**Comment by di72nn** [comment]
"Unknown error" notifications should be expandable. The text from the expanded notification may be enough, but the logcat is preferable.

**Comment by nsteinmetz** [comment]
@tcitworld if by server you mean the webapp, 2.1.2 in a docker container (wallabag/wallabag) with nginx on top of it. Waiting to the PR to be merged to test with 2.1.3 docker container.

I don"t have a rooted phone, nor android studio yet, so logs will wait :-/

**Comment by nsteinmetz** [comment]
Same results with 2.1.3 :-(

I can synchronise from wallabag server to android app but on the other side, it always fails :-(

**Comment by tcitworld** [comment]
If it downloads correctly articles from the server but can't sync changes back to the server, it's an username/password issue, whereas the rss token works properly.

**Comment by nsteinmetz** [comment]
Hmm nope:
* I changed my credentials and validated them by fetching rss token & co. It works
* I restarted the app
* I restarted firefox 
* I tried so save a link => failed.

Another try with "accepting all sll certs" ° + same process: failed.

Another try with "disable ssl" ° + same process: failed.

It worked well so far. I have a Moto 4G / Android 6.0.1 ; It worked previously with Wallabag v2 and Android App till a few weeks ago.

**Comment by Strubbl** [comment]
So adding a link via the app does not work? Can you have a look into the menu, there should be a menu entry being called "sync offline changes". Use that to upload your changes made in the app if that is present?

Just to besure: Can you check your user name for upper and lower case letters? We had an issue in the past where the wallabag core login allows case-insensitive user name, but the app does not.

**Comment by di72nn** [comment]
> **"Unknown error" notifications should be expandable. The text from the expanded notification may be enough**, but the logcat is preferable.

**Comment by nsteinmetz** [comment]
@di72nn from my point of view, it's not expandable. When you tap on the notification, nothing happens.

@Strubbl : 
*no, I can't add a link from the app or from firefox (with share > bag it)
* When I do sync offline changes, it fails too
* Credentials are lowercase - but I use exlamation marks in my password. Could it prevent working ? Like it should be espaced to prevent being interpreted somehow ?

**Comment by nsteinmetz** [comment]
... password with just letters and numbers does not change anything.

**Comment by di72nn** [comment]
@nsteinmetz https://www.youtube.com/watch?v=a6dzMzklEd4

**Comment by nsteinmetz** [comment]
@di72nn yes, I know this, it works with other applications. For this notirification in particular, it does not extend. At the beginning I was not even about to know that it was linked with wallabag. It's when I enabled background sync that I made the link with the bug.

Looking on wallabag side, I see this when I try to sync and what lead to errors:

``
wallabag_1  | ::ffff:172.18.0.1 - - [06/Nov/2016:09:35:27 +0000] "GET /bookmarklet?url=https://t.co/rjTzeRnoeK HTTP/1.0" 500 4253 "-" "okhttp/3.4.1"
```

It leads to this url: http://community.hpe.com/hpeb/attachments/hpeb/JapanEnterpriseTopics/198/1/PostgreSQL%209.6%20New%20Features%20en%2020160606-1.pdf

Itried to bag it from FF extensions and it failed too. Trying manually from Wallabag in the brower,  have:

```
500: Internal Server Error

Attempted to call function "gzuncompress" from the global namespace.
```

I just purged the database and... it works again !

I should have thought first to look on server side logs but I thought that it did not even reach the server with this error message.

Maybe there should be something like: link <url> could not be saved and also a counter like after X tries, then link is dropped to avoid  blocking others ?

**Comment by di72nn** [comment]
The original issue is fixed in `1.11.0`.
